### PR TITLE
chore:  Setup workspace for specific docSite actions

### PR DESCRIPTION
**Pull request by benjaminParisel**
* Create a doc-site folder which will contain all the specific actions for the docSite
* Mutualize the configuration to use the same dependencies for all the actions

### CODE REVIEW

File: .github/workflows/build-pr-actions-doc-site.yml [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
     paths:
-      - 'packages/pr-antora-content-guidelines-checker/*'
+      - 'packages/doc-contribs/*'
+      - '!packages/doc-contribs/*.md'
   pull_request:
     branches: [ main ]
     paths:
-      - 'packages/pr-antora-content-guidelines-checker/*'
+      - 'packages/doc-contribs/*'
+      - '!packages/doc-contribs/*.md'
```
Comment (tbouffard) [inline .github/workflows/build-pr-actions-doc-site.yml:13]: **suggestion:** include also in sub directories
```suggestion
      - '!packages/doc-contribs/**/*.md'
```

File: packages/doc-contribs/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Documentation site action
```
Comment (tbouffard) [inline packages/doc-contribs/README.md:1]: **suggestion:** 
```suggestion
# Actions for Documentation Contributions
```

File: packages/doc-contribs/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+This workspace contains the documentation site action.
```
Comment (tbouffard) [inline packages/doc-contribs/README.md:3]: **suggestion:** or something else
```suggestion
This workspace contains the actions used to help contributors of the Bonitasoft Documentation.
```

**Review by tbouffard — APPROVED** [review]
LGTM, good job 💯 
I propose few minor changes

**Comment by benjaminParisel** [comment]
Allow us to get a test report like this screenshot

Tested and validated with https://github.com/bonitasoft/actions/pull/135/checks?check_run_id=30038352468
TestResult

**Pull request merged**